src/main/java/gov/nasa/pds/tools/label/LocationValidator.java

```diff
@@ -226,8 +226,10 @@ public void validate(ValidateProblemHandler problemHandler, URL url) {
 			ruleContext.setCrawler(crawler);
 			ruleContext.setRule(rule);
 
+            LOG.info("validate:Submitting task {} to taskManager location {} rule {} ",task,location, rule.getCaption());
             LOG.debug("validate:Submitting task to taskManager location {} rule {} ",location, rule.getCaption());
             taskManager.submit(task);
+            LOG.info("validate:Returning from task {} to taskManager location {} rule {} ",task,location, rule.getCaption());
```

Comment:
@qchaupds can we also make this a DEBUG message?

Reply:
The info logging has been removed for next commit.